### ISSUE TITLE
MinLength triggers require

### ISSUE THREAD

**Issue by viktorivanov** [kazupon/vue-validator#60]
I think that the minLength validator should not trigger require on a field if not set explicitly.
For example I want to have a field with a minLength of 3 characters but It should not be required (and only if the user types something).

**Comment by kazupon** [comment]
Thank you for your comment !!

I want to understand more about your above comment. 
I hope that you provide the example codes.

**Comment by viktorivanov** [comment]
Hi,
I've made a [jsfiddle](http://jsfiddle.net/viktorivanov/b21e7d33/2/) to explain myself.

**Comment by kazupon** [comment]
Thank you for your explain jsfiddle code !!

I understood your posted issues.
I'm developing the vue-validator v2.0 now.
I'll improve the this issue at the next version.

**Comment by viktorivanov** [comment]
That's great. Thank you @kazupon .

**Comment by boris-glumpler** [comment]
Any news on this? It was kinda tripping me up just now. Validation rules should only be triggered if the field in question is required or if it's not empty.
It's very common to have more than 1 validation rule per field and currently the built-in rules are not usable because of this issue.

Maybe I've got this wrong, but shouldn't it be as easy as only validating if there actually is something to validate, so e.g.:

```
Vue.validator('rule', function(val) {
  if(isEmpty(val)) {
    return true; // No value, so bail
  }

  return /.../.test(val); // There's a value, so run whatever test we need
});
```
